src/pslLint/tblcolDoc.ts

```diff
@@ -0,0 +1,40 @@
+import * as path from 'path';
+import { Range } from '../parser/tokenizer';
+import { Diagnostic, DiagnosticSeverity, FileDefinitionRule } from './api';
+
+/**
+ * Checks whether table and columns are created with documentation.
+ */
+export class TblColDocumentation extends FileDefinitionRule {
+
+	report(): Diagnostic[] {
+		const baseName = path.basename(this.profileComponent.fsPath);
+
+		const diagnostics: Diagnostic[] = [];
+
+		const charcterOffset = this.profileComponent.textDocument.match(/^}/m).index;
```

Comment:
`match()` can possibly return `null`. Also `characterOffset` may be `undefined`. How can you handle these situations?

Reply:
Handled as discussed